metecho/api/migrations/0112_github_models_populate.py

```diff
@@ -0,0 +1,102 @@
+# Generated by Django 4.0.5 on 2022-07-12 22:21
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
```

Comment:
This is a hand-crafted migration that ensures we correctly handle existing GH users in `Project.github_users`,  `Epic.github_users`, and `Task.assigned_qa/dev`. Could use as much review as possible when it comes to running the migration in a real DB because the test suite won't cover it